moler/config/__init__.py

```diff
@@ -64,13 +64,15 @@ def configs_are_same(config_list, config_to_find):
     return False
 
 
-def load_config(config=None, from_env_var=None, config_type='yaml'):
+def load_config(config=None, from_env_var=None, **kwargs):
     """
-    Load Moler's configuration from config file
+    Load Moler's configuration from config file.
 
-    :param config: either dict or config filename directly provided (overwrites 'from_env_var' if both given)
+    config_type parameter is deprached and since 1.4.0 not need to provide. Type of config is autodetecting.
+
+    :param config: either dict or config filename directly provided (overwrites 'from_env_var' if both given).
+    :type config: dict or str
     :param from_env_var: name of environment variable storing config filename
-    :param config_type: 'dict' ('config' param is dict) or 'yaml' ('config' is filename of file with YAML content)
```

Comment:
You are removing `config_type` parameter so you should not mention it in documentation - that confuses users.
Deprecation should be put into `CHANGELOG.md` as:
```
### Deprecated
* `config_type` parameter of `load_config()` is not needed, configuration type is autodetected
```
So, as docstring I would put:
```
 """
    Load Moler's configuration from config file.
    :param config: either dict or config filename directly provided (overwrites 'from_env_var' if both given).
    :type config: dict or str
    :param from_env_var: name of environment variable storing config filename (file is in YAML format)
    :return: None
"""
```

Reply:
**kwargs is not enought. What if someone use load_config and pass to it arguments positional without parameter_name?

Reply:
15d257514b4964fd6bb7612543ec8460f3b02295 and 8d14ba8e68c42cdbae7e0dae8e0d7113e3fad0ab